### PR TITLE
HIPP-1534: CYA

### PR DESCRIPTION
**Pull request by PaulCDurham**
https://jira.tools.tax.service.gov.uk/browse/HIPP-1534

### CODE REVIEW

File: app/views/application/accessrequest/RequestProductionAccessView.scala.html [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+        <div class="hip-request-apis">
+            @for(applicationApi <- selectedApis) {
+                <details class="hip-details hip-expander">
+                    <summary class="hip-details__summary">
+                        <span class="hip-details__summary-text govuk-heading-s govuk-!-margin-bottom-0">
+                            @applicationApi.apiTitle
+                        </span>
+                    </summary>
+                    <div class="hip-details__text"> <!-- hip-details__terms -->
```
Comment (rdawson-hmrc) [inline app/views/application/accessrequest/RequestProductionAccessView.scala.html:60]: remove HTML comment?

File: conf/messages.en [hunk 2]
```diff
@@ -404,14 +404,16 @@ addAnApiSuccess.advice.body2 = View your application: {0}
...
+requestProductionAccess.heading = Check your answers before continuing
+requestProductionAccess.details = Details
+requestProductionAccess.application.name = Application name
 requestProductionAccess.instruction = Review and accept the policies for each API you plan to use in production.
-requestProductionAccess.declaration.accept = I confirm that I have read and meet the requirements outlined in the usage policies.
-requestProductionAccess.apiName = API name:
-requestProductionAccess.viewDetails = View details
-requestProductionAccess.scopes = These end points use the following scopes:
+requestProductionAccess.complete.heading = Complete your production access request
+requestProductionAccess.complete.p1 = Once the API has been added to your application you will be able to use it in the lower environments.
```
[the remark this answers is not in the document]
Comment (PaulCDurham) [reply conf/messages.en:412]: The wording is as per prototype. We'd need to raise that with Jono and Segun.

File: app/views/application/accessrequest/RequestProductionAccessView.scala.html (hunk 1 above)
Comment (PaulCDurham) [reply app/views/application/accessrequest/RequestProductionAccessView.scala.html:60]: Whoops, I'll fix that. Good spot.

**Review by rdawson-hmrc — APPROVED** [review]

File: conf/messages.en (hunk 2 above)
[the remark this answers is not in the document]
Comment (PaulCDurham) [reply conf/messages.en]: Changed

**Pull request merged**